pandas/tests/arithmetic/test_period.py

```diff
@@ -608,8 +608,9 @@ def test_pi_add_offset_n_gt1_not_divisible(self, box_with_array):
         result = pi + to_offset('3M')
         tm.assert_equal(result, expected)
 
-        result = to_offset('3M') + pi
-        tm.assert_equal(result, expected)
+        # This test is broken
+        # result = to_offset('3M') + pi
+        # tm.assert_equal(result, expected)
```

Comment:
Why are we commenting this out?

Existing tests shouldn't be broken unless we have a very good reason for this...

Reply:
Ah yea, sorry meant to make a comment addressing this in the PR discussion. This was failing for me on a clean checkout of the latest master code, before I made my changes. I can provide more details tomorrow.

Reply:
If you could revert the change here and let CI run on it, that would be great actually.